src/components/AddNewBookForm/_AddNewBookForm.scss

```diff
@@ -1,89 +1,112 @@
-.submit-book {
-  margin-top: $padding-xs;
-  width: 166px;
-  height: 47px;
-  border-radius: 19px;
-  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5);
-  background-color: $black;
-  font-size: 15px;
-  color: $white;
-  text-align: center;
-}
-
-.input-description {
-  height: 100px;
-  width: 100%;
-}
+$description-height: 200px;
 
-.add-book {
-  width: 100%;
+.upload-progress.redirect {
+  transition: all 0.3s;
   display: flex;
-  margin-left: auto;
-  margin-right: auto;
-  justify-content: space-between;
-  flex-wrap: wrap;
-  align-items: flex-start;
-}
-
-.add-book-container {
-  flex: 100%;
-  margin-top: $padding-sm;
-  float: left;
-  margin-left: auto;
-  margin-right: auto;
-
-  @include media-breakpoint(sm) {
-    flex: 50%;
-    float: left;
-    margin-left: auto;
-    margin-right: auto;
+  justify-content: center;
+  align-content: center;
+  position: fixed;
+  left: 30%;
```

Comment:
You could of course use percentages here, but I am not a big fan of this approach.
I like paddings, margins and sizes to be consistent and therefore usually just use more media queries with different fixed widhts/heights/etc.

Regarding the time, however, I think this is fine for now :)

Reply:
This should be alway in the center of the screen. Thus, the percentage is necessary.